### ISSUE TITLE
Feedback on POIAnalytics & Cartographer Journey

### ISSUE THREAD

**Issue by prastut** [prastut/foamviz#64]
Please give feedback on the following visualizations: 
* POIAnalytics Viz: https://foamviz.surge.sh/#/poi-analytics
* Journey of a Cartographer Viz: https://foamviz.surge.sh/#/cartographer-journey/

Sample of cartographer journey: https://foamviz.surge.sh/#/cartographer-journey/0xe2d128323cf7560a6e7a82726d7b425aedc7a556

## Pre-notes
The time slider on Cartographer Journey is a little buggy. Fixes are en-route. 

### POIAnalytics 
inset feedback for https://foamviz.surge.sh/#/poi-analytics

### Cartographer Journey
inset feedback for https://foamviz.surge.sh/#/cartographer-journey

**Comment by prastut** [comment]
Caleb's previous feedback: 

## POIAnalytics Viz

- [x]  Mobile UX fixes.

**Comment by prastut** [comment]
My feedback: 

## POIAnalytics Viz
Problem: Users can pan around a little but they more likely to try different cities out. Currently viz doesn't provide any way to jump from city to city. 

Possible solution: Search bar to be added to POIAnalytics Viz similar to https://hoodmaps.com/noida-neighborhood-map so that people can easily search for a particular city/area. 

Problem: Code fetches new points, every time user finishes panning around -> which leads to jerky UX. 

Possible Solution: 
Debouncing API requests #66 


## Cartographer Journey

### Problem 1: 
Long ask for the user's to go and figure out the cartographer's address, and then enter cartographer's ethereum address on https://foamviz.surge.sh/#/cartographer-journey. 

Possible solutions: 
Do a "Journey of top cartographers" inspired from
![image](https://user-images.githubusercontent.com/10279686/69438389-f6691e00-0d6a-11ea-9272-b43fc4740716.png)
https://flightmapper.io/

So that users can instantly get some gratification with top cartographers. +1  to integrate with FOAM Tools specific cartographer page. 

### Problem 2: 
Currently, the user is unable to make the journey as I had envisioned where the user will be able to hop from one point to another to get a feel of the cartographer's journey. 

Need to think more about how can this be made possible. 
Ideal UX: https://www.youtube.com/watch?v=MWyBgudQqsI

### Problem 3: 
Reduce loading time for viz. 
For the same history data load, foam.tools is atleast 2x faster. 
https://foamviz.surge.sh/#/cartographer-journey/0xe2d128323cf7560a6e7a82726d7b425aedc7a556

https://foam.tools/#/cartographers/0xe2d128323cf7560a6e7a82726d7b425aedc7a556

### Problem 3
Tooltip information needs to become clearer to give end-user clarity around what each arc represents.

**Comment by calebsheridan** [comment]
## POIAnalytics Viz

#### Problem

The height/color of columns seems based on the overall highest density hexagon (in NY).

Cities other than NY appear to have a low density of points that makes understanding how POI are mapped across individual cities harder.

#### Suggestion

Show a relative density from the map's bounding box.(Especially since global density will be handled in another viz).

**Comment by prastut** [comment]
@calebsheridan Absolutely. For this edge case, I am not sure how to go about handling UX. Still thinking, let me know you have any ideas. 

Edge case:
1. User loads initial viewport. Points are rendered. Min max is decided from the points fetched. 
2. User pans to change viewport. Now in this viewport, new points are feched and they contain a a max which is higher than the max from points fetched in the previous viewport. In that case what should happen?

Should the already rendered points height reduce due to a different max? That would look like the previously rendered towers height reducing and the new tower's height increasing.
